cgminer.c

```diff
@@ -4327,6 +4327,29 @@ void roll_work(struct work *work)
 	work->id = total_work_inc();
 }
 
+void roll_work_ntimes(struct work *work, int ntimes)
```

Comment:
I suggest we change the function to `void roll_work_ntime(struct work *work, int noffset)`, so that it will look like this `static void modify_ntime(char *ntime, int noffset)`, ntime is a field of block header.